packages/kolme-cli/src/main.rs

```diff
@@ -20,6 +24,19 @@ enum Cmd {
     },
     /// Send a transaction via an API server.
     SendTx(SendTxOpt),
+    /// Fork height information
```

Comment:
Maybe clarify what the point of this a bit more, e.g.:

```suggestion
    /// Find the first block that has the given chain version
```

Reply:
Updated, thanks!